sdk/servicebus/Microsoft.Azure.ServiceBus/tests/MessageTests.cs

```diff
@@ -260,5 +260,25 @@ public async Task MessagePropertiesShouldSupportValidPropertyTypes()
                 Assert.IsType<TimeSpan>(receivedMsg.UserProperties["TimeSpan"]);            
             });
         }
+
+        [Fact]
+        [DisplayTestMethodName]
+        public void MessageCloneShouldCreateIndependentUserProperty()
+        {
+            var messageBody = Encoding.UTF8.GetBytes("test");
```

Comment:
nit:  You may also want to do a reference comparison and prove that they are not the same dictionary reference.